src/EllCrv/Torsion.jl

```diff
@@ -564,7 +571,7 @@ function torsion_structure(E::EllCrv{nf_elem})
     structure = [fmpz(k1), fmpz(k2)]
     gens = [T1, T2]
   end
-  return [structure, gens]
+  return Vector{}[structure, gens]
```

Comment:
```suggestion
  return structure, gens
```
You can just return a tuple.